README.md

```diff
@@ -988,6 +989,52 @@ targets:
     releaseRepoName: 'unity'
 ```
 
+### Maven central (`maven`)
+
+PGP signs and publishes packages to Maven Central.
+
+**Environment**
+
+| Name                     | Description                      |
+| ------------------------ | -------------------------------- |
+| `OSSRH_USERNAME`         | Username of Sonatype repository. |
+| `OSSRH_PASSWORD`         | Password of Sonatype repository. |
+| `MAVEN_CENTRAL_USERNAME` | Username of Maven Central.       |
+| `MAVEN_CENTRAL_PASSWORD` | Password of Maven Central.       |
```

Comment:
sonatype repo and maven central are the same, so ideally only one configuration for user and pass

Reply:
the env. var. name could be also generic as it depends on `mavenRepoUrl`.
for example, mavenRepoUrl could be pointing somewhere else hence the name OSSRH or MAVEN_CENTRAL won't make sense, it would still work though.
maybe MAVEN_USERNAME and MAVEN_PASSWORD only